### PR TITLE
DownsterNet - Created plugin for new hoster downster.net

### PR DESCRIPTION
**Pull request by ill1boy**
Hey there, I hope you are doing well.
As I could not find a documentation how to properly contribute a plugin I just create a PR now.
Also I could not find a clear hint which branch is now the development one but I guess the master ist just super outdated and stable is the correct :)

So here I created a plugin for the new multihoster [Downster.net](https://downster.net).

Can you tell me how the distribution of the pyload changes happen? Is there even an update mechanism or does the user have to pull the latest sources?
Do you may also have some other advice for the Plugin? It works locally but may I'm missing something you know it should be there :)

### CODE REVIEW

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- module/plugins/hoster/DownsterNet.py  4
- module/plugins/accounts/DownsterNet.py  4
         

Clones added
============
- module/plugins/hoster/DownsterNet.py  4
- module/plugins/accounts/DownsterNet.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5379600&bid=17639805)

File: module/plugins/accounts/DownsterNet.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+import hashlib
```
Comment (vuolter) [inline module/plugins/accounts/DownsterNet.py:3]: Codacy Issue found: [Unused import hashlib](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5379600)

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- module/plugins/hoster/DownsterNet.py  4
- module/plugins/accounts/DownsterNet.py  4
         

Clones added
============
- module/plugins/hoster/DownsterNet.py  4
- module/plugins/accounts/DownsterNet.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5379600&bid=17639805)

**Comment by GammaC0de** [comment]
It seems that you are affiliated with Downster.net, am I right?

If so, can I get a test account because I can see few issues in your implementation..

**Comment by ill1boy** [comment]
Sure. Just register and you will have premium access during the beta phase :)

**Comment by ill1boy** [comment]
If you don't want to register I created the user test@pyload.net with the pw asdasd for you.

**Comment by GammaC0de** [comment]
> Which kind of issues do you see?

Mostly cosmetic / code style

**Comment by ill1boy** [comment]
I see. I'm not a python engineer so most likely there are some bad practices in there

**Comment by ill1boy** [comment]
Thank zou for merging this. For learning purpose I would be curious to know what zou changed?

**Comment by GammaC0de** [comment]
You can compare my files (https://github.com/pyload/pyload/commit/e95e89522944fb9967b2c1e3ef1a578787a37ddb) with yours.

**Pull request closed without merging**